website_product_delivery_method/views/customer_group_view.xml

```diff
@@ -0,0 +1,55 @@
+<?xml version="1.0"?>
```

Comment:
Can you check if `encoding="utf-8"?` doesn't have to be explicitly included in 10.0?

Reply:
I think it is actually related to XML version, it should be UTF-8 or UTF-16 by default if the encoding is not explicitly defined